docs/release-notes/feature-node-selectors.rst

```diff
@@ -0,0 +1,14 @@
+:orphan:
+
+**New Features**
+
+-  Kubernetes Configuration: Allow cluster administrators to define Determined resource pools on
+   Kubernetes in terms of node selectors and/or affinities. Clusters administrators configure
+   resource pool's node selectors and/or affinities on the default pod spec level, under the
+   task_container_defaults.cpu_pod_spec or task_container_defaults.gpu_pod_specs. In this way,
+   cluster administrators can split a single cluster into several resource pools by way of node
+   labels.
+
+-  WebUI: For Determined clusters deployed on Kubernetes with multiple resource pools defined in
+   terms of node selectors and/or affinities, the resource pool slot counts correctly reflect the
+   state of the entire cluster. Slot counts and scheduling respect node selectors and affinities.
```

Comment:
awkward sentence:

Slot counts and scheduling respect node selectors and affinities.